src/main/java/com/icthh/xm/ms/entity/service/impl/FunctionServiceImpl.java

```diff
@@ -79,40 +83,73 @@ public FunctionContext execute(final String functionKey,
 
         Map<String, Object> vInput = CustomCollectionUtils.emptyIfNull(functionInput);
 
+        dynamicPermissionCheckService.checkContextPermission(DynamicPermissionCheckService.FeatureContext.FUNCTION,
+            "XMENTITY.FUNCTION.EXECUTE", functionKey);
```

Comment:
Maybe better move in constant or in the configuration?

Reply:
if You mean privilege name "XMENTITY.FUNCTION.EXECUTE", yes. I will do it on phase 2

Reply:
Fixed